warehouse/models/staging/gtfs/_stg_gtfs.yml

```diff
@@ -98,3 +98,84 @@ models:
       - name: bearing
       - name: odometer
       - name: speed
+  - name: stg_gtfs_schedule__routes
+    description: |
+      Cleaned GTFS schedule route data.
+      See https://gtfs.org/schedule/reference/#routestxt for specification.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - base64_url
+            - ts
+            - route_id
+    columns:
+      - name: base64_url
+      - name: ts
+      - name: route_id
+        description: '{{ doc("gtfs_routes__route_id") }}'
+        tests:
+        - not_null
+      - name: agency_id
+        description: '{{ doc("gtfs_routes__agency_id") }}'
+      - name: route_short_name
+        description: '{{ doc("gtfs_routes__route_short_name") }}'
+      - name: route_long_name
+        description: '{{ doc("gtfs_routes__route_long_name") }}'
+      - name: route_desc
+        description: '{{ doc("gtfs_routes__route_desc") }}'
+      - name: route_type
+        description: '{{ doc("gtfs_routes__route_type") }}'
+        tests:
+        - not_null
+      - name: route_url
+        description: '{{ doc("gtfs_routes__route_url") }}'
+      - name: route_color
+        description: '{{ doc("gtfs_routes__route_color") }}'
+      - name: route_text_color
+        description: '{{ doc("gtfs_routes__route_text_color") }}'
+      - name: route_sort_order
+        description: '{{ doc("gtfs_routes__route_sort_order") }}'
+      - name: continuous_pickup
+        description: '{{ doc("gtfs_routes__continuous_pickup") }}'
+      - name: continuous_drop_off
+        description: '{{ doc("gtfs_routes__continuous_drop_off") }}'
+  - name: stg_gtfs_schedule__trips
+    description: |
+      Cleaned GTFS schedule trip data.
+      See https://gtfs.org/schedule/reference/#tripstxt for specification.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - base64_url
+            - ts
+            - trip_id
+          severity: warn
```

Comment:
My theory was: this is an error on every table to start, until it actually occurs, then you downgrade to warn and add a dedupe intermediate step and re-promote to error. (i.e., what this is saying is that this is not a guarantee on this stage any more because we've observed a violation and we can't fix it before this step)

Reply:
works for me